lib/augmentConfig.js

```diff
@@ -235,8 +235,10 @@ function mergeConfigs(a, b) {
 		}
 
 		if (b.plugins) {
-			pluginMerger.plugins = [...new Set(pluginMerger.plugins.concat(b.plugins))];
+			pluginMerger.plugins = pluginMerger.plugins.concat(b.plugins);
 		}
+
+		pluginMerger.plugins = [...new Set(pluginMerger.plugins)];
```

Comment:
ditto

Reply:
I also agree with @Mouvedia. We can pass `lib/__tests__/applyOverrides.test.js` even if using `lib/augmentConfig.js` before this change.